ape_etherscan/explorer.py

```diff
@@ -37,7 +38,16 @@ def get_contract_type(self, address: str) -> Optional[ContractType]:
         except JSONDecodeError:
             return None
 
-        return ContractType.parse_obj({"abi": abi, "contractName": source_code.name})
+        contract_type = ContractType.parse_obj({"abi": abi, "contractName": source_code.name})
+        if source_code.name == "Vyper_contract" and "symbol" in contract_type.view_methods:
+            try:
+                checksummed_address = self.provider.network.ecosystem.decode_address(address)
+                contract = self.create_contract(checksummed_address, contract_type=contract_type)
```

Comment:
The more I think about it, the more I think `create_contract` is a confusing name. Maybe `get_contract` or something

Reply:
Agreed. I can add this to my list of breaking-changes for next time